scalableminds/webknossos
# Persist organization name to NML

### URL of deployed dev instance (used for testing):
- https://nmlpersistorganization.webknossos.xyz

### Steps to test:
- download nml (both via backend and frontend)
- `experiment` tag should include `organization` attribute
- if you have read access to multiple datasets, the one of the organization with a matching name should be picked on re-upload

### Issues:
- fixes #3536

------
